src/main/java/org/usfirst/frc4904/robot/RobotMap.java

```diff
@@ -31,32 +67,101 @@
     public static class Metrics {
         public static class Chassis {
             public static final double TICKS_PER_REVOLUTION = -1; // TODO: CHANGE CONSTS
-            public static final double DIAMETER_INCHES = -1;
-            public static final double CIRCUMFERENCE_INCHES = Metrics.Chassis.DIAMETER_INCHES * Math.PI;
-            public static final double TICKS_PER_INCH = Metrics.Chassis.TICKS_PER_REVOLUTION
-                    / Metrics.Chassis.CIRCUMFERENCE_INCHES;
+            public static final double DIAMETER_METERS = Units.inchesToMeters(5);
+            public static final double CIRCUMFERENCE_METERS = Metrics.Chassis.DIAMETER_METERS * Math.PI;
+            public static final double TICKS_PER_METER = Metrics.Chassis.TICKS_PER_REVOLUTION
+                    / Metrics.Chassis.CIRCUMFERENCE_METERS;
             public static final double DISTANCE_FRONT_BACK = -1;
-            public static final double DISTANCE_SIDE_SIDE = -1;
-            public static final double INCHES_PER_TICK = Metrics.Chassis.CIRCUMFERENCE_INCHES
-                    / Metrics.Chassis.TICKS_PER_REVOLUTION;
+            public static final double DISTANCE_TRACK_WIDTH = -1;
+            public static final double METERS_PER_TICK = 1 / TICKS_PER_METER;
         }
     }
 
     public static class PID {
         public static class Drive {
+            public static final double P = -1;
+            public static final double I = -1;
+            public static final double D = -1;
+            public static final double F = -1;
+            public static final double TOLERANCE = -1;
+            public static final double D_TOLERANCE = -1;
         }
 
         public static class Turn {
+            public static final double P = -1;
+            public static final double I = -1;
+            public static final double D = -1;
+            public static final double F = -1;
+            public static final double TOLERANCE = -1;
+            public static final double D_TOLERANCE = -1;
         }
 
     }
 
+    public static class DriveConstants {// TODO: Change Constants
+        // Field Carpet
+        // public static final double ksVolts = 0.0018;
+        // public static final double kvVoltSecondsPerMeter = 4.9;
+        // public static final double kaVoltSecondsSquaredPerMeter = 0.184;
+        // public static final double kTrackwidthMeters = .61; //0.5842
+        // public static final double kPDriveVel = 6.27;
+        // School Carpet
+        public static final double VOLTS = -1;
+        public static final double VOLT_SECONDS_PER_METER = -1;
+        public static final double VOLT_SECONDS_SQUARED_PER_METER = -1;
+        public static final double TRACK_WIDTH_METERS = -1; // 0.5842 // 0.6063751884752512
```

Comment:
repeated from `Metrics.Chassis`. remove one of them (though maybe we want to use the experimental one over the one that we measure? Though I've found their always super close, so that's a separate discussion. for now, you could probably leave it like this